soh/include/functions.h

```diff
@@ -495,6 +495,8 @@ void func_80031B14(PlayState* play, ActorContext* actorCtx);
 void func_80031C3C(ActorContext* actorCtx, PlayState* play);
 Actor* Actor_Spawn(ActorContext* actorCtx, PlayState* play, s16 actorId, f32 posX, f32 posY, f32 posZ,
                    s16 rotX, s16 rotY, s16 rotZ, s16 params);
+Actor* Actor_Spawn_NoRandomize(ActorContext* actorCtx, PlayState* play, s16 actorId, f32 posX, f32 posY, f32 posZ,
+                   s16 rotX, s16 rotY, s16 rotZ, s16 params);
```

Comment:
i'm not the biggest fan of this naming convention (it feels like it'd be nicer to call `Actor_Spawn_Randomize` when randomizing instead, but i understand the reasoning behind it (fewer places where things need to be changed)

maybe it'd be nicer if it was `Actor_Spawn_NeverRandom` instead?

Reply:
Yeah I've struggled with the name for this function myself honestly. The original function name has to stay as Actor_Spawn because it's reliant on all the vanilla game spawns basically, so that'd be a lot of places to rename it otherwise as you said.

I'm OK with renaming it to `Actor_Spawn_NeverRandom`, or maybe `Actor_Spawn_NotRandom`?

Reply:
Changed it to Actor_Spawn_NotRandom now, but I'm open for other names still.